src/crystal/system/unix/epoll/event_loop.cr

```diff
@@ -213,12 +212,8 @@ class Crystal::Epoll::EventLoop < Crystal::EventLoop
   end
 
   def interrupt : Nil
-    # the atomic makes sure we only write 1 byte to the pipe, so we can assume
-    # there's only 1 byte to read when the pipe fd is triggered
-    return unless @interrupted.test_and_set
-
-    byte = 1_u8
-    LibC.write(@pipe[1], pointerof(byte), 1)
+    # the atomic makes sure we only write once
```

Comment:
Is writing more than once a problem when using eventfd? Could be an alternative to simply write multiple times and not have the atomic variable at all. The eventfd will just return the sum of values that has been written to it, after all (unless using the semaphore flag). 

(BTW, this method is to interrupt the blocking wait, right? Sometimes the term `notify` is used for that)

Reply:
You're right. It's not needed as for the pipe (we can write 0xfffffffffffffffe times until it blocks) but it helps avoid pointless `write` syscalls, so :balance_scale: ?